cypress/e2e/customerRegistration.js

```diff
@@ -31,7 +31,8 @@ describe("Tests for customer registration", () => {
   it("should register customer", { tags: ["@customer", "@stagedOnly"] }, () => {
     const email = `${startsWith}${faker.datatype.number()}@example.com`;
```

Comment:
What do you think about that?
```suggestion
    const email = `${startsWith}${faker.datatype.number()}@example.com`;
    const registrationLinkRegex = /\[(\s*http[^\]]*)\]/;
    
    customerRegistration({ email, channel: defaultChannel.slug });
    getMailActivationLinkForUser(email)
    getMailActivationLinkForUser(email, registrationLinkRegex)
```